packages/transaction-controller/src/TransactionControllerIntegration.test.ts

```diff
@@ -108,6 +109,13 @@ const newController = async (options: any) => {
 };
 
 describe('TransactionController Integration', () => {
+  let clock: SinonFakeTimers | undefined;
+
+  afterEach(() => {
+    clock?.restore();
+    clock = undefined;
```

Comment:
[nit] instead of this we could add a beforeEach:
```
 let clock: sinon.SinonFakeTimers;
  beforeEach(() => {
    clock = useFakeTimers();
  });

  afterEach(() => {
    clock.restore();
  });
  ```

Reply:
we aren't using fake timers in all of our tests, but i'll see if we can

Reply:
done https://github.com/MetaMask/core/pull/3826/commits/da8c9e09124bfdf7165d983d4c08290e24628acf